content/04.network_applications.md

```diff
@@ -0,0 +1,43 @@
+## Knowledge Graph Applications
+
+1. Mention that these graphs can be used for discovery
+2. Mention representation learning (aka representing a graph as dense vectors for nodes and/or edges)
+3. 
+
```

Comment:
You have both techniques and applications here. What do you think about talking about the same literature but from two perspectives. You'd have a subsection at the `###` level called "Unifying Techniques" and another called "Unifying Applications". This would help readers approach the literature from either angle. In both of these subsections, you'd approach the same papers.